packages/graphback-codegen-client/src/templates/gqlTemplates.ts

```diff
@@ -25,26 +25,26 @@ export const findOneQuery = (t: ModelDefinition) => {
   const fieldName = getFieldName(t.graphqlType.name, GraphbackOperationType.FIND_ONE)
 
   return `query ${fieldName}($id: ${t.primaryKey.type}!) {
-    ${fieldName}(id: $id) {
-      ...${t.graphqlType.name}ExpandedFields
-    }
-  }`
+            ${fieldName}(id: $id) {
```

Comment:
Why indentation was changed

Reply:
Okay should I only change the closing bracket with respect to 'q' of query?